### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by janhavisinghh**
Change log update for demo release

### CODE REVIEW

**Review by msasikanth — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
 
 ## Next Release
 ### Internal
+- [In Progress: 11 Nov 2020] Migrate app screens to use ViewBinding
```
Comment (msasikanth) [inline CHANGELOG.md:5]: We are already tracking this with ` - [In Progress: 30 Nov 2020] Migrate app to use ViewBinding `. So, please remove this.

**Review by msasikanth — CHANGES_REQUESTED** [review]
Left one comment. PTAL

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,25 +2,25 @@
 
 ## Next Release
 ### Internal
+- [In Progress: 30 Nov 2020] Migrate app to use ViewBinding
+- [In progress: 01 Dec 2020] Migrate `ShortCodeSearchResultScreen` to Mobius
+- Add PR comment commands for running instrumented tests and rebasing PR
```
Comment (msasikanth) [inline CHANGELOG.md:7]: Can you recheck the rebase operation, please? It doesn't match the current `DEMO` PR.

https://github.com/simpledotorg/simple-android/blob/release/2020-12-14/CHANGELOG.md

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,25 +2,25 @@
...
 - Add breadcrumbs for different stages in search operations
 
 ### Changes
-- Updated translations: `pa-IN`, `te-IN`, `ti-ET`, `bn-IN`
+- Updated translations: `pa-IN`, `te-IN`, `ti-ET`, `bn-IN`, `mr-IN`
```
Comment (msasikanth) [inline CHANGELOG.md:11]: Same comment as above

**Review by msasikanth — CHANGES_REQUESTED** [review]
Left couple of comments. PTAL

File: CHANGELOG.md (hunk 2 above)
Comment (janhavisinghh) [reply CHANGELOG.md:7]: Hey, I have added the Demo release things match in the Demo section. I added the rest of the new changes in the Next release section.

Comment (msasikanth) [reply CHANGELOG.md:7]: Oh, wait, yeah this correct. My bad

**Review by msasikanth — APPROVED** [review]

**Pull request merged**